src/app/static/src/app/store/metadata/actions.ts

```diff
@@ -23,18 +25,21 @@ export const actions: ActionTree<MetadataState, RootState> & MetadataActions = {
     },
 
     async getReviewInputMetadata(context) {
-        const { commit, rootState, state } = context;
+        const { commit, rootState } = context;
         const iso3 = rootState.baseline.iso3;
         const sap = rootState.surveyAndProgram;
         const fileTypes = [FileType.Shape]
         if (!sap.ancError && sap.anc) {
-            fileTypes.push(FileType.ANC)
+            fileTypes.push(FileType.ANC);
+            addAreaLevelsToSAPData(rootState, "anc", commit);
         }
         if (!sap.programError && sap.program) {
-            fileTypes.push(FileType.Programme)
+            fileTypes.push(FileType.Programme);
+            addAreaLevelsToSAPData(rootState, "program", commit);
         }
         if (!sap.surveyError && sap.survey) {
-            fileTypes.push(FileType.Survey)
+            fileTypes.push(FileType.Survey);
+            addAreaLevelsToSAPData(rootState, "survey", commit);
```

Comment:
Do we have to worry at all about how much we're saving into local storage here? Or is that not a problem?

Reply:
this just replaces them in state, so we should be using basically the same amount of memory right?

Reply:
Oh sorry, I wasn't looking at this properly. Was thinking this was writing area hierarchy into the data for each of these. But that isn't the case.